holopy/core/io/io.py

```diff
@@ -131,15 +130,18 @@ def load_image(inf, spacing=None, illum_wavelen=None, medium_index=None, illum_p
         else:
             arr = arr[:, :, channel]
     elif channel is not None and channel > 0:
-        warnings.warn("Warning: not a color image (channel number ignored)")
+        warn("Warning: not a color image (channel number ignored)")
 
```

Comment:
Does load image do something smart with color channels and xarray? I don't think so yet, but it would be a good thing to add.